selfdrive/car/subaru/carcontroller.py

```diff
@@ -64,7 +65,7 @@ def update(self, CC, CS, now_nanos):
         can_sends.append(subarucan.create_preglobal_es_distance(self.packer, cruise_button, CS.es_distance_msg))
 
     else:
-      if pcm_cancel_cmd and (self.frame - self.last_cancel_frame) > 0.2:
+      if pcm_cancel_cmd and (self.frame - self.last_cancel_frame)*DT_CTRL > 0.2:
```

Comment:
this should be done in a separate PR, it changes behavior

Reply:
done